src/Organizations/OrganizationForm/OrganizationSummaryForm/OrganizationSummaryForm.js

```diff
@@ -143,6 +174,22 @@ function OrganizationSummaryForm({ initialValues }) {
           ))}
         </Field>
       </Col>
+      <Col
+        xs={6}
+        md={3}
+      >
+        <FieldMultiSelectionFinal
```

Comment:
FieldOrganizationTypes is outsourced

Reply:
It seems that `<FieldCode orgId={initialValues.id} />` causes this errror when creating a new organization element:
_Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'active')_
Can please somebody check that?

Reply:
Or does that comment mean that I just should use only ONE query/hook for getting ALL organization-types (without the ids)?
_additionally org types are dictionary data that used on several screens (form, details, filters), it means that they can be loaded on module level and cached instead of fetching it each time (especially in batch when we don't really expect a lot of types in one org)_
If this should be the case, do you have example where you load data on module level and pass on to the functions?

Reply:
adding `useTypes` to `OrganizationDetailsContainer.test.js` throws this error:
`TypeError: (0 , _hooks.useTranslatedCategories) is not a function`
Any idea where it is coming from?